spec/models/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
```

Comment:
you can use let instead of having global variables: e.g
```ruby
let(:user) { build(:user) }
subject { user }
````

Reply:
```ruby
before do 
  subject { build(:user) }
end
```
It will achieve the same result

Reply:
Ok, I change my global variables for these lines:

```
  let(:user) { build(:user) }
  subject { user }
```